tools/list_to_ucptrie/README.md

```diff
@@ -0,0 +1,86 @@
+# CodePointTrie Builder Tool
```

Comment:
Is this still required?

Reply:
Good catch; this code lives elsewhere now (in the `icu_codepointtrie_builder` crate). I'll remove it from this PR.